searcher/src/main/java/ru/otus/searcher/applicationRunner/CityLoadingService.java

```diff
@@ -0,0 +1,40 @@
+package ru.otus.searcher.applicationRunner;
+
+import dto.CityDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.otus.searcher.controllers.CitiesSearchController;
+import ru.otus.searcher.converter.CityDtoToCityEntityConverter;
+import ru.otus.searcher.entity.City;
+import ru.otus.searcher.repository.CityRepository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
```

Comment:
Good point - logging is important part.
Let you start using it everywhere and write logs.